configman/config_manager.py

```diff
@@ -234,9 +278,26 @@ def __init__(
             # 'app_name' from the parameters passed in, if they exist.
             pass
 
-        if use_auto_help and self._get_option('help').value:
-            self.output_summary()
-            admin_tasks_done = True
+        try:
+            if use_auto_help and self._get_option('help').value:
+                self.output_summary()
+                admin_tasks_done = True
+        except NotAnOptionError:
+            # the current command-line implementation already has a help
+            # mechanism of its own that doesn't required the use of a
+            # option in configman.  This error is ignorable
+            pass
+
+        # keys that end with a "$" are called "blocked_by_sufix".
```

Comment:
"blocked_by_suffix" (lacking f). Trivial but it'd make it hard to do a grep or replace-all